WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -689,6 +709,32 @@ public void onPostChanged(OnPostChanged event) {
     public void onPostUploaded(OnPostUploaded event) {
         if (isAdded() && event.post.getLocalSiteId() == mSite.getId()) {
             loadPosts(LoadMode.FORCED);
+
+            final PostModel post = event.post;
+            boolean isDraft = post != null && PostStatus.fromPost(post) == PostStatus.DRAFT;
+            if (isDraft) {
+                if (PostUtils.isPublishable(post)) {
```

Comment:
This check isn't necessary, we check if a post is publishable (has at least one of content, title, and excerpt) before publishing, and wouldn't make a `PUSH_POST` request if it didn't qualify.

(If we make a mistake in the future and upload something unpublishable, we'll get an error response from the server, which we should show.)

Along with removing this, we can also pull the `event.isError()` case above the `isDraft` check since it's the same whether `isDraft` or not.